src/Sparrow/Json/Parsing/UnmanagedJsonParser.cs

```diff
@@ -8,17 +8,9 @@ namespace Sparrow.Json.Parsing
 {
     public sealed unsafe class UnmanagedJsonParser : IJsonParser
     {
-        private static readonly byte[] NaN = { (byte)'N', (byte)'a', (byte)'N' };
-
-        private static readonly byte[] PositiveInfinity =
-        {
-            (byte)'I', (byte)'n', (byte)'f', (byte)'i', (byte)'n', (byte)'i', (byte)'t', (byte)'y'
-        };
-
-        private static readonly byte[] NegativeInfinity =
-        {
-            (byte)'-', (byte)'I', (byte)'n', (byte)'f', (byte)'i', (byte)'n', (byte)'i', (byte)'t', (byte)'y'
-        };
+        private static readonly byte[] NaN = "NaN"u8.ToArray();
+        private static readonly byte[] PositiveInfinity = "Infinity"u8.ToArray();
+        private static readonly byte[] NegativeInfinity = "-Infinity"u8.ToArray();
```

Comment:
This is duplicated from the text writer, it also isn't `ReadOnlySpan` so cannot be put on the data segment?
Why are those plain arrays?

Reply:
because the code that uses it requires it to be an array instead of a  `ReadOnlySpan<byte>` that's the only reason.